README.md

```diff
@@ -18,10 +18,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
This is related with the following issue: https://github.com/GoogleChrome/lighthouse-ci/issues/172